esx_service/utils/auth_api.py

```diff
@@ -190,7 +195,8 @@ def modify_privileges(privileges, allow_create, volume_maxsize_in_MB, volume_tot
     logging.debug("modify_privileges: allow_create=%s, volume_maxsize_in_MB=%s, volume_totalsize_in_MB=%s",
                   allow_create, volume_maxsize_in_MB, volume_totalsize_in_MB)
     if allow_create is not None:
-        if allow_create == "True":
+        # allow_create has been valided. It is either "true" or "false"
+        if allow_create == "true":
```

Comment:
Not always. Consuming tenant_access_set through admin cli gives allow_create as string, and through VMODL it might be boolean or string.